source/_includes/installation/operating_system.md

```diff
@@ -212,7 +216,7 @@ Use this method only if Method 1 does not work for you.
 9. Select **Flash!** to start writing the image.
    - If the operation fails, decompress the .xz file and try again.
 ![Screenshot of the Etcher software showing the Flash button highlighted.](/images/installation/etcher5.png)
-1.  When Balena Etcher has finished writing the image, you will see a confirmation.
+   1.  When Balena Etcher has finished writing the image, you will see a confirmation.
```

Comment:
this looks odd with a number, should it be a bullet point?

Reply:
Good point. I stumbled across that as well because that weird numbering is there in the current guide. So I thought there might have been a reason for that. I personally am fine with changing that.